client/src/app/components/CreateProject/styles.css

```diff
@@ -40,20 +40,30 @@
 }
 
 .form-wrapper button {
-  padding: 5px;
-  margin: 10px;
+  border: none;
+  height: 35px;
+  width: 75px;
+  padding: 8px;
+  margin: 2px;
+  background: #ec5252;
+  border: 1px solid #e8e9eb;
+  color: white;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: bold;
+  cursor: pointer;
+  float: right;
+}
+
+.form-wrapper button:hover {
+  background: #b33b3b;
```

Comment:
Optional at this point, but I had introduced variables for colours, to try to reuse and share them across the app.